joblib/_ray.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import print_function, division, absolute_import
+import ray
+from .parallel import AutoBatchingMixin, ParallelBackendBase
+
+class RayBackend(ParallelBackendBase, AutoBatchingMixin):
+    """
```

Comment:
BTW, style can help a lot: 

this is the guide most use https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html